components/src/dynamo/sglang/llm_engine.py

```
@@ -405,6 +409,151 @@ async def generate(
+                return {"status": "ok", "message": "Memory already released"}
+            try:
+                await controller.quiesce(tags)
+                return {
+                    "status": "ok",
+                    "message": (
+                        f"Memory released for tags: {tags}"
+                        if tags is not None
+                        else "Memory released"
+                    ),
+                }
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)